cargo-miri/bin.rs

```diff
@@ -903,6 +903,13 @@ fn phase_rustdoc(fst_arg: &str, mut args: env::Args) {
         show_error(format!("cross-interpreting doc-tests is not currently supported by Miri."));
     }
 
+    // Doc-tests of `proc-macro` crates (and their dependencies) are always built for the host,
+    // so we are not able to run them in Miri.
+    if ArgFlagValueIter::new("--crate-type").any(|crate_type| crate_type == "proc-macro") {
+        eprintln!("Running doc-tests of `proc-macro` crates is not currently supported by Miri.");
```

Comment:
```suggestion
        eprintln!("Running doctests of `proc-macro` crates is not currently supported by Miri.");
```
https://doc.rust-lang.org/rustdoc/documentation-tests.html spells it without a "-"

Reply:
The preexisting `show_error()` above uses "doc-tests" (with "-"). 🙃

Reply:
Ah, we should probably fix that too, then.^^ Sorry for this.
(I can do that in a separate PR.)